packages/client/src/redux/TrecipeList/action.ts

```diff
@@ -42,6 +45,9 @@ export const createTrecipeRequest = (trecipeData: CreateNewTrecipeDTO): AppThunk
     return (dispatch: ThunkDispatch<RootState, unknown, Action<string>>) => {
         TrecipeService.createTrecipe(trecipeData).then((createdTrecipe: Trecipe) => {
             dispatch(addTrecipe(createdTrecipe));
+            UserService.getUser().then((updatedUser: UserResponse) => {
```

Comment:
Just curious (because I'm sure this is here for a reason lol).
What's the purpose of setting user on every request?

Reply:
Future proofing lol. Each user has a Trecipe array that contains the Trecipe id they owned. I am setting the user whenever a Trecipe is added/deleted/duplicated in order to update the user's Trecipe array in the redux store. 

Buttt I don't think we are using this Trecipe array anywhere in our code for now haha,  so maybe we should remove it? or maybe comment it out 😛

Reply:
Ahh I see. I think it’s fine to keep it there. I have a few endpoints I defined but not using at all as well haha. Maybe we could comment those during our doc & code comment improvement issue.